restapi/src/main/java/io/stargate/web/docsapi/service/JsonConverterService.java

```diff
@@ -0,0 +1,290 @@
+package io.stargate.web.docsapi.service;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.node.*;
+import io.stargate.db.datastore.Row;
+import io.stargate.db.schema.Column;
+import io.stargate.web.docsapi.dao.DocumentDB;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import org.apache.commons.lang3.tuple.ImmutablePair;
+
+public class JsonConverterService {
```

Comment:
So copying the stuff form the `DocumentService` is a good first step. A second should be imo to:

* refactor that `convertToJsonDoc` into smaller junks that would make it easier to understand
* check for what shared logic between methods can be extracted/unified
* add docs to the methods so that reader can understand what's going on